docs/Using-SRIOV-and-Clear-Containers.md

```diff
@@ -0,0 +1,218 @@
+# Setup to use SRIO-V with Clear Containers
+
+Single Root IO Virtualization enables splitting a physical device into
```

Comment:
- IO or I/O which is more common?

This sentence feels a bit unwieldy to me. How about we split it into something like:

> Single Root I/O Virtualization enables splitting a physical input output device into multiple separate slices, or 
> virtual functions (VFs). This enables direct pass through to virtual machines or containers.